app/controllers/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+ class UsersController < ApplicationController
+
+  def new
+    redirect_to home_path if current_user
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to login_path
+    else
+      render :new
+    end
+  end
+
+  private
+
+    def user_params
```

Comment:
Which one? `user_params` ? I thought that private methods were indented two spaces past the `private` ?